R/aes-evaluation.r

```diff
@@ -2,39 +2,105 @@
 #'
 #' Most aesthetics are mapped from variables found in the data. Sometimes,
 #' however, you want to delay the mapping until later in the rendering process.
-#' ggplot2 has three stages of the data that you can map aesthetics from. The
-#' default is to map at the beginning, using the layer data provided by the
-#' user. The second stage is after the data has been transformed by the layer
-#' stat. The third and last stage is after the data has been transformed and
-#' mapped by the plot scales. The most common example of mapping from stat
-#' transformed data is the height of bars in [geom_histogram()]:
-#' the height does not come from a variable in the underlying data, but
-#' is instead mapped to the `count` computed by [stat_bin()]. An example of
-#' mapping from scaled data could be to use a desaturated version of the stroke
-#' colour for fill. If you want to map directly from the layer data you should
-#' not do anything special. In order to map from stat transformed data you
-#' should use the `after_stat()` function to flag that evaluation of the
-#' aesthetic mapping should be postponed until after stat transformation.
-#' Similarly, you should use `after_scale()` to flag evaluation of mapping for
-#' after data has been scaled. If you want to map the same aesthetic multiple
-#' times, e.g. map `x` to a data column for the stat, but remap it for the geom,
-#' you can use the `stage()` function to collect multiple mappings.
+#' ggplot2 has three stages of the data that you can map aesthetics from, and
+#' three functions to control at which stage aesthetics should be evaluated.
 #'
+#' @usage # These functions can be used inside the `aes()` function
+#' # used as the `mapping` argument in layers.
```

Comment:
Since you're add a fake usage, maybe also include some example `aes()` calls in comments?